rebar.config.script

```diff
@@ -0,0 +1,13 @@
+case <<1:16/native>> of
+    <<1,0>> ->
+        CONFIG;
+    <<0,1>> -> 
+        CXXFLAGS = [ { "CXXFLAGS", "$CXXFLAGS -DWORDS_BIGENDIAN" } ],
+        P = lists:keyfind( port_env, 1, CONFIG ),
+        case P of
```

Comment:
you can actually avoid temp variable `P` by using `list:keyfiend` expression directly in case.